app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -47,13 +61,196 @@ class AppointmentRepositoryAndroidTest {
+        PatientAddress(
+            address1,
+            null,
+            faker.address.city(),
+            faker.address.state(),
+            "India",
+            Instant.now(),
+            Instant.now()
+        )
+    )
+    val patient1 = UUID.randomUUID()
+    val date1 = LocalDate.now().minusDays(100)
+    val bp1 = UUID.randomUUID()
+    database.patientDao().save(
+        Patient(
+            patient1,
+            address1,
+            faker.name.name(),
+            faker.name.name(),
+            Gender.FEMALE,
+            LocalDate.parse("1947-08-15"),
+            null,
+            PatientStatus.ACTIVE,
+            Instant.now(),
+            Instant.now(),
+            SyncStatus.DONE
+        )
+    )
+    database.bloodPressureDao().save(listOf(
+        BloodPressureMeasurement(
+            uuid = bp1,
+            systolic = 190,
+            diastolic = 100,
+            syncStatus = SyncStatus.PENDING,
+            userUuid = testData.qaUserUuid(),
+            facilityUuid = testData.qaUserFacilityUuid(),
+            patientUuid = patient1,
+            createdAt = Instant.now(),
+            updatedAt = Instant.now()
+        )
+    ))
+
+    val patient2 = UUID.randomUUID()
+    val address2 = UUID.randomUUID()
+    database.addressDao().save(
+        PatientAddress(
+            address2,
+            null,
+            faker.address.city(),
+            faker.address.state(),
+            "India",
+            Instant.now(),
+            Instant.now()
+        )
+    )
+    database.patientDao().save(
+        Patient(
+            uuid = patient2,
+            addressUuid = address2,
+            fullName = faker.name.name(),
+            searchableName = faker.name.name(),
+            gender = Gender.TRANSGENDER,
+            dateOfBirth = LocalDate.parse("1997-08-15"),
+            age = null,
+            status = PatientStatus.ACTIVE,
+            createdAt = Instant.now(),
+            updatedAt = Instant.now(),
+            syncStatus = SyncStatus.DONE
+        )
+    )
+
+    val patient3 = UUID.randomUUID()
+    val address3 = UUID.randomUUID()
+    val date3 = LocalDate.now().minusDays(10)
+    val bp30 = UUID.randomUUID()
+    val bp31 = UUID.randomUUID()
+    database.addressDao().save(
+        PatientAddress(
+            address3,
+            null,
+            faker.address.city(),
+            faker.address.state(),
+            "India",
+            Instant.now(),
+            Instant.now()
+        )
+    )
+    database.patientDao().save(
+        Patient(
+            patient3,
+            address3,
+            faker.name.name(),
+            faker.name.name(),
+            Gender.MALE,
+            LocalDate.parse("1977-11-15"),
+            null,
+            PatientStatus.MIGRATED,
+            Instant.now(),
+            Instant.now(),
+            SyncStatus.DONE
+        )
+    )
+    database.bloodPressureDao().save(listOf(
+        BloodPressureMeasurement(
```

Comment:
Will you move these dummy object creations to `TestData` in your next commit? This test looks gigantic otherwise right now.

Reply:
If not the next commit, then at some point. I'll add this to icebox.